docs/releases/v1.7.1.md

```diff
@@ -5,6 +5,7 @@ Welcome to the latest release of `OPA` module of [Kubernetes Fury Distribution](
 This is a patch release including the following changes:
 
 - Updated the provided Grafana dashboard to use the new metrics from Gatekeeper.
+- Added TIgera operator namespaces to the default exemptions in custom rules.
```

Comment:
```suggestion
- Added Tigera operator namespaces to the default exemptions in custom rules.
```

Reply:
I've fixed it at the same time you  commented 😄